src/main/target/BEESTORM/config.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+#include <platform.h>
+
+#ifdef USE_TARGET_CONFIG
+
+#include "common/axis.h"
+#include "common/maths.h"
+
+#include "config/feature.h"
+
+#include "drivers/light_led.h"
+#include "drivers/pwm_esc_detect.h"
+
+#include "fc/config.h"
+#include "fc/controlrate_profile.h"
+#include "fc/rc_modes.h"
+#include "fc/rc_controls.h"
+
+#include "flight/mixer.h"
+#include "flight/pid.h"
+
+#include "pg/vcd.h"
+
+#include "rx/rx.h"
+
+#include "io/serial.h"
+#include "io/osd.h"
+
+#include "sensors/battery.h"
+#include "sensors/gyro.h"
+
+#include "telemetry/telemetry.h"
+
+#define BEST_FRSKY_RSSI_CH_IDX     9
+
+#ifdef BRUSHED_MOTORS_PWM_RATE
+#undef BRUSHED_MOTORS_PWM_RATE
+#endif
+
+#define BRUSHED_MOTORS_PWM_RATE 32000           // 32kHz
```

Comment:
This should be set through `motorConfigMutable()->dev.motorPwmRate = BRUSHED_MOTORS_PWM_RATE`.

Reply:
So does this mean that you just copied the other target without looking into what they are doing in detail? This is not how this works, these targets are not meant to be 'fire and forget' by their owners, but owners are expected to keep maintaining them after they have been added.